lib/galaxy/datatypes/binary.py

```diff
@@ -2407,6 +2407,13 @@ def get_type(self):
         return "Sciex WIFF/SCAN archive"
 
 
+class Tensorflow(Binary):
```

Comment:
Still need to remove this class.

Reply:
ping @astrovsky01, such a class is not needed as you specify subclass="True" in the datatypes_conf file.

Reply:
This class still needs to be removed @astrovsky01, this isn't resolved yet.

Reply:
@mvdbeek I was waiting while I discussed with Bjorn the security aspects of the checkpoint files, as they contain code that is run inside tensorflow.